src/main/kotlin/io/arrowkt/service/InterestCalculation.kt

```diff
@@ -1,8 +1,10 @@
 package io.arrowkt.service
 
+import arrow.data.EitherTPartialOf
 import arrow.data.Kleisli
+import arrow.effects.ForIO
 
 interface InterestCalculation<Account, Amount> {
 
-    fun calculateInterest(): Kleisli<Valid<Amount>, Account, Amount>
```

Comment:
This used to be much nicer when we had:

```
typealias Valid<A> = EitherT<ForIO, AccountServiceException, A>

typealias AccountOperation<A> = Kleisli<Valid<A>, AccountRepository, A>
```
But when I introduced `EitherTPartialOf` I had to get rid of the `Valid`

any ideas how to improve it?

Reply:
make an alias for it too

Reply:
I need to better explain the problem.

`Valid<A>` was used also for `ReportOperation<A>` in `ReportingService` and a few other places.... 

I could use `Valid<A>` for different types.

Now instead of `Valid` I have `EitherTPartialOf<ForIO, AccountServiceException>` and this only has the Left type, no Right type. So I cannot reuse it for different types of `A`. 

Can we somehow use the `Valid<A>` typealias and also get the Monad?